test/unit/commands/general-specs.js

```diff
@@ -120,6 +120,13 @@ describe('General', function () {
       await driver.getWindowSize().should.be.equal('size');
     });
   });
+  describe('getWindowRect', function () {
+    it('should get window size', async () => {
+      sandbox.stub(driver.bootstrap, 'sendAction')
+        .withArgs('getDeviceSize').returns({width: 300, height: 400});
+      await driver.getWindowRect().toString().should.be.equal({width: 300, height: 400, x: 0, y: 0}.toString());
```

Comment:
toString() will return something like `[object Object]`. You need to wrap objects into JSON.stringify to reach proper results